pymess/models/dialer.py

```diff
@@ -39,6 +40,7 @@ class AbstractDialerMessage(BaseMessage):
         ('HANGUP_BY_CUSTOMER', _('answered - hangup by customer'), 13),
         ('ERROR', _('error'), 66),
         ('DEBUG', _('debug'), 77),
+        ('ERROR_NOT_SENT', _('error essage was not sent'), 88),
```

Comment:
esssage -> message... but to give more sense, I suggest "error during sending of message"